src/fmu/sumo/explorer/objects/_child_collection.py

```diff
@@ -16,6 +17,7 @@
     "fmu.case.user.id",
     "fmu.realization.id",
     "fmu.iteration.name",
+    "fmu.iteration.parameters",
```

Comment:
Since this will increases the payload size returned from Sumo by quite a lot, maybe it shouldn't be included in the default property list? We can get the parameter list with a request using SumoClient instead of extracting it from a `Table` instance.

```python

# AggregatedTable.parameters could do this:

query = {
    "size": 1,
    "_source": ["fmu.iteration.parameters"],
    "query": {
        "bool": {
            "must": [
                { ..filters }
            ]
        }
    }
}

res = sumo.post("/search", json=query)
parameters = res.json()["hits"]["hits"][0]
```